service/health/response.go

```diff
@@ -17,11 +17,18 @@ type ClusterStatus struct {
 
 // NodeStatus holds data about the health/status of a node.
 type NodeStatus struct {
-	Name         string `json:"name"`
-	Ready        bool   `json:"ready"`
-	IP           string `json:"ip"`
-	Hostname     string `json:"hostname"`
-	InstanceType string `json:"instance_type"`
-	CPUCount     int64  `json:"cpu_count"`
-	MemoryBytes  int64  `json:"available_memory_bytes"`
+	Name                   string `json:"name"`
+	Ready                  bool   `json:"ready"`
+	IP                     string `json:"ip"`
+	Hostname               string `json:"hostname"`
+	InstanceType           string `json:"instance_type"`
+	AvailabilityRegion     string `json:"availability_region"`
+	AvailabilityZone       string `json:"availability_zone"`
+	KubeletVersion         string `json:"kubelet_version"`
+	OperatorVersion        string `json:"operator_version"`
+	CPUCount               int64  `json:"cpu_count"`
+	MemoryCapacityBytes    int64  `json:"memory_capacity_bytes"`
+	MemoryAllocatableBytes int64  `json:"memory_allocatable_bytes"`
+	EphemeralStorageCap    int64  `json:"ephemeral_storage_capacity_bytes"`
+	EphemeralStorageAvail  int64  `json:"ephemeral_storage_allocatable_bytes"`
```

Comment:
What is the sorting logic here? Is it sorted like it is returned from `kubectl`?

Reply:
Nothing super formal, these come from all over the place in `kubectl` anyway but my sorting is roughly "what node am I? what can I do? what am I doing?"